src/databricks/labs/ucx/source_code/known.json

```diff
@@ -12653,6 +12653,12 @@
     "openai.types.shared_params.function_parameters": [],
     "openai.version": []
   },
+  "opencensus": {
+    "opencensus": []
+  },
+  "opencensus-context": {
+    "opencensus": []
```

Comment:
how come that two different libraries "own" the same root package?

Reply:
opencensus-context is a smaller focused subset of opencensus.